libretroshare/src/util/extaddrfinder.cc

```diff
@@ -145,7 +145,7 @@ void* doExtAddrSearch(void *p)
 	
 	std::vector<std::string> res ;
 
-	ExtAddrFinder *af = (ExtAddrFinder*)p ;
+	ExtAddrFinder *af = reinterpret_cast<ExtAddrFinder*>(p);
```

Comment:
use static cast for void*->type* (https://stackoverflow.com/questions/310451/should-i-use-static-cast-or-reinterpret-cast-when-casting-a-void-to-whatever)

Reply:
Done